src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1546,6 +1558,20 @@ groovyscript.wiki.pyrotech.kiln.failureChance.value=Sets the chance to fail the
 groovyscript.wiki.pyrotech.kiln.failureOutput.value=Sets the output when the recipe failed
 groovyscript.wiki.pyrotech.kiln.add=Adds recipes in the format `name`, `input`, `output`, `burnTime`, `failureChance`, `failureOutput`
 
+groovyscript.wiki.pyrotech.oven.note0=Stone and Refractory Oven includes some recipes from the Furnace Registry that can't be removed here
```

Comment:
i think this should be worded similar to how `groovyscript.wiki.chisel.carving.note` is - perhaps
```
You cannot add or remove recipes added via the Furnace Registry here, you have to use `furnace.add` or `furnace.remove` to manipulate those recipes directly.
```
i dont think theres a need to specify the `Stone and Refractory Oven` part